outlines/structure.md

```diff
@@ -0,0 +1,56 @@
+# Application structure and code style
+
+1. What's different between Meteor and other kinds of apps?
+    1. All JavaScript, enables code sharing
+    2. Structure is flexible
+    3. Currently everything is about magic file names
+    4. App scope vs. package scope vs. file scope variables
+2. Client vs. server vs. both
+3. Smooth path from prototype to production, three sizes of apps, S, M, L
+4. General stuff
```

Comment:
Well they have to if they are going to use `meteorhacks:npm` no?

I take Mitar's point about encouraging package maintainers to do it, but I think the idea of every NPM package being wrapped by a well maintained Meteor package has been thoroughly debunked. The important ones maybe thanks to heroism on the part of @dandv and others, but I think the future doesn't look like that.